src/plugins/instrument_nodejs.js

```diff
@@ -269,53 +269,27 @@ class InstrumentNodejs {
 
     _shouldTrace(tracer, url) {
```

Comment:
Was this section updated manually or by `eslint --fix`?

Reply:
This was updated manually to be in line with the recent changes to instrument_fetch file: https://github.com/lightstep/lightstep-tracer-javascript/pull/217/files#diff-80d87308d219969a5b4dcc3b01bf7351L241

Reply:
And same with `instrument_xhr` below :D